priv/eleveldb_multi.schema

```diff
@@ -117,16 +117,14 @@
   "multi_backend.$name.leveldb.compression",
   "riak_kv.multi_backend", [
   {default, on},
-  {datatype, flag},
-  hidden
+  {datatype, flag}
 ]}.
 
 {mapping,
   "multi_backend.$name.leveldb.compression.algorithm",
   "riak_kv.multi_backend", [
-  {default, lz4},
-  {datatype, {enum, [snappy, lz4]}},
-  hidden
+  {new_conf_value, lz4},
+  {datatype, {enum, [snappy, lz4]}}
 ]}.
 
 %% @see leveldb.delete_threshold
```

Comment:
Does anyone know why (or whether) a translation is needed (or not) in the _multi version of the schema?